src/structures/Utils.ts

```diff
@@ -32,8 +33,13 @@ export abstract class TrackUtils {
   }
 
   static setTrackPartial(partial: string[]): void {
-    if (!Array.isArray(partial) || !partial.every(str => typeof str === "string"))
-      throw new Error("Provided partial is not an array or not a string array.");
+    if (
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585532)
```suggestion
    if (
```